scripts/bootstrap_vm.sh

```diff
@@ -188,3 +188,28 @@ if [ "${NESSUS_INSTALL}" = "true" ]
 then
   install_nessus "${NESSUS_SERVER}" "${NESSUS_KEY}" "${NESSUS_GROUPS}"
 fi
+
+# Check if the OS is RHEL 7
+if [[ "$OS_TYPE" == *"Red Hat Enterprise"* && "$OS_TYPE" == *"7."* ]]; then
+    echo "This is Red Hat Enterprise Linux 7."
+    
+    # Register the system and attach a subscription pool
+    subscription-manager register --org=7324337 --activationkey=Rhel-els
+
+
+    # Refresh subscription-manager and verify identity
+    subscription-manager refresh
+    subscription-manager identity
+
+    # Install insights-client and register it
+    yum install -y insights-client
+    insights-client --register
+
+    # Enable repositories
+    subscription-manager config --rhsm.manage_repos=1
+    subscription-manager repos --enable rhel-7-server-els-rpms
```

Comment:
Doesn't this need some kind of auth?

Reply:
Nope, I don't think it does because we can include certificates inside the box. We just need the activation key name and org id.